srcpkgs/OpenCL-Headers/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'OpenCL-Headers'
+pkgname=OpenCL-Headers
+version=2023.04.17
+revision=1
+build_style=cmake
+short_desc="OpenCL 3.0.14 (Open Computing Language) header files"
+maintainer="Andrew J. Hesford <ajh@sideband.org>"
+license="Apache-2.0"
+homepage="https://github.com/KhronosGroup/OpenCL-Headers"
+distfiles="https://github.com/KhronosGroup/OpenCL-Headers/archive/v${version}.tar.gz"
+checksum=0ce992f4167f958f68a37918dec6325be18f848dee29a4521c633aae3304915d
+provides="OpenCL-Headers-${version}_${revision}"
+replaces="OpenCL-Headers>=0"
```

Comment:
```suggestion
provides="opencl-headers-${version}_${revision}"
replaces="opencl-headers>=0"
```
These lines replace an old, removed `opencl-headers` package, so they need to be carried forward from the `opencl2-headers` template verbatim.